Examples/Io/Root/include/ActsExamples/Io/Root/RootTrackSummaryWriter.hpp

```diff
@@ -67,6 +67,8 @@ class RootTrackSummaryWriter final : public WriterT<ConstTrackContainer> {
     std::string fileMode = "RECREATE";
     /// Switch for adding full covariance matrix to output file.
     bool writeCovMat = false;
+    /// Write GSF specific things (for now only some material statistics)
+    bool writeGsfSpecific = false;
```

Comment:
@AJPfleger this would also be an option instead of the `-1` for the `nUpdates` branch.

Reply:
Ah you changed this already.